File: sample/java/com/example/java/NestedIfElse.java
package com.example.java;

public class NestedIfElse {
    public void foo() {
        if (true) {
            return;
        }
        System.out.println();
        System.out.println();
    }

    public void bar() {
        if (true) {
            System.out.println();
        } else if (false) {
            if (false) {
                System.out.println();
            }
        } else {
            System.out.println();
            System.out.println();
        }
    }

    public void baz() {
        try {
            if (true) {
                System.out.println();
                System.out.println();
            }
        } catch (RuntimeException e) {
            System.out.println();
        }
    }
}

File: sample/kotlin/com/example/kotlin/NestedIfElse.kt
package com.example.kotlin

class NestedIfElse {
    fun foo() {
        if (false) {
            return
        }
        if (false) {
            return
        }
        println()
        println()
    }

    fun bar() {
        if (true) {
            println()
        } else if (false) {
            println()
        } else {
            println()
        }
    }

    fun baz(): Int {
        println()
        return if (true) {
            println()
            0
        } else {
            0
        }
    }

    fun qux() {
        if (true) {
            try {
                if (false) {
                    println()
                    println()
                }
            } catch (e: Exception) {
                if (false) {
                    println()
                    println()
                }
            }
        }
        println()
    }
}
